lua/pymple/utils.lua

```diff
@@ -105,6 +105,37 @@ function M.to_import_path(module_path)
   return result
 end
 
+---Converts a path to a python reference path
+---@param module_path string: The path to a python module
+---@param python_symbol string | nil: The symbol to reference
+---@return string: The python reference path
+function M.to_python_reference_path(module_path, python_symbol)
+  local result, _ =
+    module_path:gsub("/", "."):gsub("%.py$", ""):gsub(".__init__", "")
+  if python_symbol then
+    result = result .. "." .. python_symbol
+  end
+  return result
+end
```

Comment:
I guess we should merge that with `M.to_import_path` just above since it's just an improved version of it.

Reply:
Thank you for the feedback and for offering to help! 🙏🏻

Since merging utils.to_python_reference_path with utils.to_import_path might require some changes in `update_imports` which I haven't used yet, I'd be happy if you take that if you're okay with it. I'll work on adding the tests for utils.to_python_reference_path, utils.to_import_statement, and other things.

Reply:
I just merged both functions and tests for `to_python_reference_path`

Reply:
I've added tests and fd as a dependency. Looks like all items are completed